src/modules/smugmug.js

```diff
@@ -0,0 +1,40 @@
+// SmugMug services
+(function(hello) {
+
+	hello.init({
+
+		smugmug: {
+			name: 'SmugMug',
+
+			oauth : {
+				version : "1.0a",
+				auth	: "http://api.smugmug.com/services/oauth/1.0a/authorize?Access=Full&Permissions=Modify",
+				//http://api.smugmug.com/services/oauth/1.0a/authorize?Access=Full&Permissions=Modify&allowThirdPartyLogin=0&showSignUpButton=false&viewportScale=1.0
+				request : "http://api.smugmug.com/services/oauth/1.0a/getRequestToken?oauth_callback=http://localhost:3000/redirect.html",
```

Comment:
Please omit the oauth_callback. The value of `oauth_callback` is derived from the `redirect_uri` at the point the OAuth1a request URI is constructed by the OAuth Proxy service.